cpp/neuralnet/nninputs.h

```diff
@@ -10,6 +10,9 @@
 #include "../game/boardhistory.h"
 #include "../game/rules.h"
 
+#define SYMMETRY_NOTSPECIFIED -1
```

Comment:
Can we move these two constants into the NNInputs namespace, and make them instead constexpr int?

Reply:
Done